docs/installation.md

```diff
@@ -39,6 +39,27 @@ KVM acceleration can be used
 
 With this kind of output, you're ready to go!
 
+#### Cloud Provider KVM supported instances
+
+If you intend to use a cloud provider to test Ignite, you can follow the instructions below.
+
+##### Amazon Web Services
+
+Amazon EC2 [bare metal instances](https://aws.amazon.com/about-aws/whats-new/2018/05/announcing-general-availability-of-amazon-ec2-bare-metal-instances/) provide direct access to the  Intel® Xeon® Scalable processor and memory resources of the underlying server. These instances are ideal for workloads that require access to the hardware feature set (such as Intel® VT-x), for applications that need to run in non-virtualized environments for licensing or support requirements, or for customers who wish to use their own hypervisor.
+
+Here's a list of instances with KVM support, with pricing (as of July 2019), to help you test Ignite.
+
+| Family | Type | Pricing (US-West-2) per On Demand Linux Instance Hr | vCPUs | Memory (GiB) | Instance Storage (GB) | EBS-Optimized | Available Network Performance | IPv6 Support |
+| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
+|Compute optimized | c5.metal | $4.08 | 96 |192 |EBS only | Yes |25 Gigabit |Yes |
```

Comment:
Maybe say that all thses instances have 25Gb network, IPv6 support and are EBS-optimized instead of duplicating that content?

Reply:
Sure, will update.

Reply:
Done